eng/tox/tox.ini

```diff
@@ -75,7 +75,7 @@ commands =
       {toxinidir}
 
 
-[testenv:lint]
+[testenv:pylint]
```

Comment:
wondering if we should make this change since it will require everyone to switch the command they are used to running locally for pylint. if we do make this change, let's update the docs too: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md#tox

Reply:
ctrl-f `tox.*lint` picked up additional changes on top of `tests.md`. Renamed everywhere in the docs as well. Pretty certain that's all of em now.